### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by BradleyBoutcher**
- Introduce CONTRIBUTING.md
- Link to general contributing guidelines

### CODE REVIEW

**Comment by BradleyBoutcher** [comment]
Please squash & merge

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+
+## Table of Contents
```
Comment (izgeri) [inline CONTRIBUTING.md]: We can probably drop the TOC for a contributing file this short

**Review by izgeri — CHANGES_REQUESTED** [review]
This is a lower prio repo so feel free to work on others first, but the Testing content in the README should be moved here too

**Comment by jtuttle** [comment]
@BradleyBoutcher Got time to merge this down soon? Looks like a pretty simple change needed to get it mergeable.

**Review by izgeri — APPROVED** [review]

**Review by izgeri — CHANGES_REQUESTED** [review]
Ack @BradleyBoutcher I approved without reading my prior message - but it looks like this is still outstanding:
- [ ] the Testing content in the README should be moved here too

**Review by izgeri — APPROVED** [review]

**Pull request merged**